src/raftstore/store/snap.rs

```diff
@@ -875,8 +873,7 @@ impl Snapshot for Snap {
         for cf_file in &self.cf_files {
             delete_file_if_exist(&cf_file.tmp_path);
             if file_exists(&cf_file.path) {
-                let mut size_track = self.size_track.wl();
-                *size_track = size_track.saturating_sub(cf_file.size);
+                self.size_track.fetch_sub(cf_file.size, Ordering::SeqCst);
```

Comment:
can fetch_sub overflow here?

Reply:
It can happen because now register a snapshot multi times is not forbidden by `SnapManager`.